src/main/resources/application-local.yml

```
@@ -83,4 +83,14 @@ sentry:
\ No newline at end of file
+  environment: local
+
+management:
+  endpoints:
+    web:
+      exposure:
+        include: health,info,prometheus,metrics,env
+
+  metrics:
+    tags:
+      application: houme
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)